File: src/_1ProgrammingBasicsWithJavaMay2023/_2ConditionalStatements/_1Lab/_07AreaOfFigures.java
package ConditionalStatementsLab;

import java.util.Scanner;

public class _07AreaOfFigures {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String typeFigure = scanner.nextLine();
        double area = 0;
        if (typeFigure.equals("square")){
            double a = Double.parseDouble(scanner.nextLine());
            area = a * a;
        } else if (typeFigure.equals("rectangle")){
            double a = Double.parseDouble(scanner.nextLine());
            double b = Double.parseDouble(scanner.nextLine());
            area = a * b;
        } else if (typeFigure.equals("circle")){
            double r = Double.parseDouble(scanner.nextLine());
            area = Math.PI * r * r;
        } else if (typeFigure.equals("triangle")){
            double a = Double.parseDouble(scanner.nextLine());
            double ha = Double.parseDouble(scanner.nextLine());
            area = (a * ha) / 2;
        }
        System.out.printf("%.3f", area);
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_2ConditionalStatements/_1Lab/_07AreaOfFigures.js
function areaOfFigures(figureType, first, second) {
    let area;
    if (figureType === "square") {
        const a = first;
        area = a * a;
    } else if (figureType === "rectangle") {
        const a = first;
        const b = second;
        area = a * b;
    } else if (figureType === "circle") {
        const r = first;
        area = r * r * Math.PI;
    } else if (figureType === "triangle") {
        const a = first;
        const h = second;
        area = (a * h) / 2;
    }
    console.log(`${area.toFixed(3)}`)
}